llarp/path/path.cpp

```diff
@@ -439,15 +456,12 @@ namespace llarp
         auto dlt = now - m_LastLatencyTestTime;
         if (dlt > path::latency_interval && m_LastLatencyTestID == 0)
         {
-          routing::PathLatencyMessage latency;
-          latency.T = randint();
-          m_LastLatencyTestID = latency.T;
-          m_LastLatencyTestTime = now;
-          SendRoutingMessage(latency, r);
-          FlushUpstream(r);
-          // reset ID so we don't mark ourself as dead if we drop a latency sample
-          r->loop()->call_later(
-              1s, [self = shared_from_this()]() { self->m_LastLatencyTestID = 0; });
+          SendLatencyMessage(r);
+          // latency test FEC
+          r->loop()->call_later(1s, [self = shared_from_this(), r]() {
```

Comment:
Perhaps this should be a little longer, say 2s?  1s is rare but possible for a path (and might be more possible if someone runs in paranoid 8-hop path mode).  2s seems rare enough to be a fine cutoff.

Reply:
yea 2s works